hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -794,6 +792,29 @@ private Result increment(final HRegion region, final OperationQuota quota,
     return r == null ? Result.EMPTY_RESULT : r;
   }
 
+  private Get toGet(final Mutation mutation) throws IOException {
```

Comment:
Looks like a candidate for `static` util method. Maybe we can move this to some Mutation related util class? This file anyways has lot of code (with more APIs, it will just increase :) )

Reply:
Thank you for your review!

I couldn't find any existing Mutation related util class. I will just add static here.